### PR TITLE
Allow user to copy sample (read-only) files packaged with module

### LINKED ISSUE #838: Add Support to get copy of samples folder
# 💡 Summary #

Repackage samples to be in ScubaGear module and provide a command to make a local copy. 

## Motivation and context ##

This would be useful because we do not wan the user to search in the installed module folder for samples. 

## Implementation notes ##

Please provide details for implementation, such as:

- Move samples folder to be in  module package
- Provide command for user to make a copy of the samples directory in specified directory (e.g., home directory)

## Acceptance criteria ##

How do we know when this work is done?

- [x] Samples are packaged with ScubaGear Module
- [x] Command to make sampe copy works

### PR DESCRIPTION
**Pull request by crutchfield**
#  #
  

## 🗣 Description ##

Adds functions, Copy-ScubaSampleReport and Copy-ScubaSampleConfigFile, to get copies of files packaged with the ScubaGear module.  

Closes #838 

## 💭 Motivation and context ##

Moving sample reports and config files into the module so support publishing ScubaGear as a single module to a public package repository, PowerShell Gallery.

## 🧪 Testing ##

- Added unit test for copy functions, all completed successfully
- Ran all unit test, all completed successfully
- Added test cases to Smoke Test for support commands, all completed successfully

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] Changes are sized such that they do not touch excessive number of files.
- [X] *All* future TODOs are captured in issues, which are referenced in code comments.
- [X] These code changes follow the ScubaGear [content style guide](https://github.com/cisagov/ScubaGear/blob/main/CONTENTSTYLEGUIDE.md).
- [X] Related issues these changes resolve are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [X] All relevant type-of-change labels added.
- [X] All relevant project fields are set.
- [X] All relevant repo and/or project documentation updated to reflect these changes.
- [X] Unit tests added/updated to cover PowerShell and Rego changes.
- [X] Functional tests added/updated to cover PowerShell and Rego changes.
- [X] All relevant functional tests passed.
- [X] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [ ] PR passed smoke test check.
- [ ] Feature branch has been rebased against changes from parent branch, as needed

  Use `Rebase branch` button below or use [this](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) reference to rebase from the command line.
- [ ] Resolved all merge conflicts on branch
- [ ] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [ ] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

File: PowerShell/ScubaGear/Modules/Support/Support.psm1 [hunk 1]
```diff
@@ -28,14 +28,119 @@ function Copy-ScubaBaselineDocument {
         New-Item -ItemType Directory -Path $Destination | Out-Null
     }
 
-    @("teams", "exo", "defender", "aad", "powerplatform", "sharepoint") | ForEach-Object {
+    @("teams", "exo", "defender", "aad", "powerbi", "powerplatform", "sharepoint") | ForEach-Object {
```
Comment (james-garriss) [inline PowerShell/ScubaGear/Modules/Support/Support.psm1:31]: This appears to be a new service in the list of service, but per Crutch it is not a new service.  This requires comments to explain what is going on.

**Review by james-garriss — APPROVED** [review]

**Comment by nanda-katikaneni** [comment]
@crutchfield are you planning any additional commits for this pr, if not, can you please resolve @Sloane4 comment and rebase. thanks.

File: PowerShell/ScubaGear/Testing/Unit/PowerShell/Support/Copy-ScubaSampleConfigFile.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,51 @@
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "../../../../Modules/Support/Support.psm1") -Function 'Copy-ScubaConfigFile' -Force
+
+InModuleScope Support {
+    Describe "Copy sample config files to specified directory" {
+        $SampleFiles = @(
```
[the remark this answers is not in the document]
Comment (crutchfield) [reply PowerShell/ScubaGear/Testing/Unit/PowerShell/Support/Copy-ScubaSampleConfigFile.Tests.ps1]: No. Removed hardcoded file names and used Get-ChildItem instead.

**Comment by crutchfield** [comment]
@nanda-katikaneni Resolved comments and rebased.  Ready to merge.

**Comment by crutchfield** [comment]
@nanda-katikaneni Ready for merge to main

**Pull request merged**